tests/integration/sugar/test_transaction.py

```diff
@@ -431,3 +436,32 @@ async def test_sign_and_submit(self, client):
         payment_transaction = Payment.from_dict(payment_dict)
         response = await sign_and_submit(payment_transaction, client, WALLET)
         self.assertTrue(response.is_successful())
+
+    @test_async_and_sync(
+        globals(),
+        [
+            "xrpl.transaction._calculate_fee_per_transaction_type",
+        ],
+    )
+    async def test_basic_calculate_fee_per_transaction_type_offline(self, client):
+        fee = await _calculate_fee_per_transaction_type(
+            Payment(
+                account="rweYz56rfmQ98cAdRaeTxQS9wVMGnrdsFp",
+                amount=IssuedCurrencyAmount(
+                    currency="USD",
+                    issuer="rweYz56rfmQ98cAdRaeTxQS9wVMGnrdsFp",
+                    value="0.0001",
+                ),
+                destination="rweYz56rfmQ98cAdRaeTxQS9wVMGnrdsFp",
+                send_max=IssuedCurrencyAmount(
+                    currency="BTC",
+                    issuer="rweYz56rfmQ98cAdRaeTxQS9wVMGnrdsFp",
+                    value="0.0000002831214446",
+                ),
+            ),
+            client,
+        )
+
+        # If FeeSettings ledger object is updated, the below assert needs to be changed
+        # correspondingly
+        self.assertEqual(fee, "200")
```

Comment:
nit: could we use the reference_fee variable from the config file here so that only that file needs to be updated in the future?

Reply:
completed in 16099ec2
I tried to use external libraries like `toml` and `configparser`. Thy could not parse the `.ci-config/rippled.cfg` file. Let me know if I can accomplish this without manually reading the file contents.